### ISSUE TITLE
How can I get all the gifs in all the categories and show them in `GiphyGridView`

### ISSUE THREAD

**Issue by sambhav2358** [Giphy/giphy-android-sdk#183]
I already know how to get the different types of gifs in different section. But what if I want all of them together without navigating ? Is it possible? Please provide the code in java if possible

**Comment by ALexanderLonsky** [comment]
The SDK does not provide this, but you can use Giphy [APIs](https://developers.giphy.com/docs/api/endpoint#trending) and implement your own solution.

**Comment by sambhav2358** [comment]
So I can just pass the media in a list and then set it to recycler view right?

**Comment by ALexanderLonsky** [comment]
yes, you can fetch medias using those endpoints, put them into a list and implement your own grid.
